packages/ui-components/src/components/DataGrid/DataGrid.tsx

```diff
@@ -1,24 +1,14 @@
 /*
  * Tupaia
- * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ * Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
  */
 import React from 'react';
-import { DataGrid as MuiDataGrid, GridRowsProp, GridColDef, GridDensity } from '@mui/x-data-grid';
-
-interface DataGridProps {
-  columns: GridColDef[];
-  rows: GridRowsProp;
-  autoPageSize?: boolean;
-  density?: GridDensity;
-}
+import { DataGrid as MuiDataGrid, DataGridProps } from '@mui/x-data-grid';
 
 export const DataGrid = ({
-  columns,
-  rows,
   autoPageSize = false,
```

Comment:
autoPageSize actually defaults to false so all this component actually does is set the density default value. I'd be pretty tempted to remove it from ui-components and just import it directly in the admin panel